frontend/src/folder/Modal/CreateFolderModal.tsx

```diff
@@ -0,0 +1,23 @@
+import { FC } from 'react';
+import ReactDOM from 'react-dom';
+import { ModalProps } from '../types/modal';
+
+const FolderModal: FC<ModalProps> = ({ onClose, children }) => {
+  return (
+    <>
+      {ReactDOM.createPortal(
```

Comment:
You can just use shadcn dialog component

In general, I'd recommend you to use more of their components

Reply:
Done!